Zend/tests/036.phpt

```diff
@@ -11,4 +11,4 @@ try {
 
 ?>
 --EXPECT--
-Cannot access offset of type object on array
+Cannot access offset of type Closure on array
```

Comment:
Only a question, not a suggestion: What do you think of also specifying what is allowed? `Array offset must be int or string, cannot access offset of type Closure`

Reply:
We can only really do this for arrays and strings :/ as objects that implement ArrayOffset can in theory support any type